src/engine/strat_engine/devlinks.rs

```diff
@@ -0,0 +1,118 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+use std::collections::HashSet;
+use std::fs;
+use std::io::ErrorKind;
+use std::os::unix::fs::symlink;
+use std::path::{Path, PathBuf};
+use std::str;
+
+use super::super::errors::EngineResult;
+
+use engine::Engine;
+
+pub const DEV_PATH: &str = "/dev/stratis";
+
+/// Set up directories and symlinks under /dev/stratis based on current
+/// config. Clear out other stuff.
```

Comment:
Maybe you want to be precise about meaning of "other stuff".

Reply:
ok